src/_shared/scss/_core.scss

```diff
@@ -182,8 +190,17 @@ $mq-breakpoints: (
 
 .justify--start { justify-content: flex-start; }
 
-// ----- Gimbap ----- //
+// Gimbap fonts
 
+@mixin f-headline {
```

Comment:
I believe Regis has removed all the font mixins here: https://github.com/guardian/commercial-templates/commit/88c509e9d4ffcd1ec81713f30a2591f1c801709c